### PR TITLE
Changing text in test.txt.

### PR DESCRIPTION
**Pull request by MathewTWilliams**
This is a comment. I changed the text in file.

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Hello!
\ No newline at end of file
+Hello!
+Hello2!
```
Comment (stantonp7) [inline test.txt:2]: Can you please add a smiley face after this !?

**Review by stantonp7 — COMMENTED** [review]
Read comments

File: test.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-Hello!
\ No newline at end of file
+Hello!
```
Comment (stantonp7) [inline test.txt:1]: Test

**Review by stantonp7 — APPROVED** [review]
LGTM

**Comment by bPaige725** [comment]
Test comment

**Pull request merged**